docs/kpod-pull.1.md

```diff
@@ -21,26 +21,26 @@ images from archives and local storage using different transports.
 ## imageID
 Image stored in local container/storage
 
-## DESTINATION
+## SOURCE
 
- The DESTINATION is a location to store container images
- The Image "DESTINATION" uses a "transport":"details" format.
+ The SOURCE is a location to get container images
+ The Image "SOURCE" uses a "transport":"details" format.
 
  Multiple transports are supported:
 
   **dir:**_path_
   An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
 
   **docker://**_docker-reference_
-  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$HOME/.docker/config.json`, which is set e.g. using `(docker login)`.
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$XDG_RUNTIME_DIR/containers/auth.json`, which is set e.g. using `(kpod login)`. An image can also be pulled from the registry without specifying the transport.
```

Comment:
I'm not getting the last added sentence here?  We're talking about authorization in this section but this sentence is talking about transport?

Reply:
Yeah, I just wanted to point out that you can pull from a registry without typing the transport, `docker://` - I can rephrase it or remove it.

Reply:
fixed